mesonbuild/compilers/mixins/gnu.py

```diff
@@ -366,6 +366,12 @@ def get_compiler_dirs(self, env: 'Environment', name: str) -> typing.List[str]:
                 return self._split_fetch_real_dirs(line.split('=', 1)[1])
         return []
 
+    def get_lto_args(self) -> typing.List[str]:
```

Comment:
Shouldn't this be `get_lto_compile_args()`? I wonder why CI doesn't fail. Are we missing a test for this codepath?

Reply:
I'd like to get away from the `_compiler_` and `_link_` in arg names. This is all prep work for a series that starts peeling the compilers and linkers apart into two separate objects so in the future we'd hopefully be able to just haver `linker.get_lots_args` and `compiler.get_lto_args`

Reply:
That makes sense. But afaict `get_lto_args()` is not getting called anywhere in this PR, so I don't understand why it's getting added here. Doesn't really affect anything right now since it's dead code only till your follow-up PR is merged, but I thought I'd make sure I understood whether it's dead code or if I missed something :)

Reply:
oh, that seems like another rebase fail then.